src/bentoml_cli/cli.py

```diff
@@ -8,52 +8,49 @@
 from bentoml_cli.models import add_model_management_commands
 from bentoml_cli.containerize import add_containerize_command
 
+from bentoml_cli.bentos import bento_commands
 
-def create_bentoml_cli() -> click.Group:
+from bentoml import __version__ as BENTOML_VERSION
+from bentoml._internal.context import component_context
 
-    from bentoml import __version__ as BENTOML_VERSION
-    from bentoml._internal.context import component_context
+component_context.component_type = "cli"
 
-    component_context.component_type = "cli"
+CONTEXT_SETTINGS = {"help_option_names": ("-h", "--help")}
 
-    CONTEXT_SETTINGS = {"help_option_names": ("-h", "--help")}
+@click.group(cls=BentoMLCommandGroup, context_settings=CONTEXT_SETTINGS)
+@click.version_option(BENTOML_VERSION, "-v", "--version")
+def bentoml_cli():
+    """
+    \b
+    ██████╗░███████╗███╗░░██╗████████╗░█████╗░███╗░░░███╗██╗░░░░░
+    ██╔══██╗██╔════╝████╗░██║╚══██╔══╝██╔══██╗████╗░████║██║░░░░░
+    ██████╦╝█████╗░░██╔██╗██║░░░██║░░░██║░░██║██╔████╔██║██║░░░░░
+    ██╔══██╗██╔══╝░░██║╚████║░░░██║░░░██║░░██║██║╚██╔╝██║██║░░░░░
+    ██████╦╝███████╗██║░╚███║░░░██║░░░╚█████╔╝██║░╚═╝░██║███████╗
+    ╚═════╝░╚══════╝╚═╝░░╚══╝░░░╚═╝░░░░╚════╝░╚═╝░░░░░╚═╝╚══════╝
+    """
 
-    @click.group(cls=BentoMLCommandGroup, context_settings=CONTEXT_SETTINGS)
-    @click.version_option(BENTOML_VERSION, "-v", "--version")
-    def bentoml_cli():
-        """
-        \b
-        ██████╗░███████╗███╗░░██╗████████╗░█████╗░███╗░░░███╗██╗░░░░░
-        ██╔══██╗██╔════╝████╗░██║╚══██╔══╝██╔══██╗████╗░████║██║░░░░░
-        ██████╦╝█████╗░░██╔██╗██║░░░██║░░░██║░░██║██╔████╔██║██║░░░░░
-        ██╔══██╗██╔══╝░░██║╚████║░░░██║░░░██║░░██║██║╚██╔╝██║██║░░░░░
-        ██████╦╝███████╗██║░╚███║░░░██║░░░╚█████╔╝██║░╚═╝░██║███████╗
-        ╚═════╝░╚══════╝╚═╝░░╚══╝░░░╚═╝░░░░╚════╝░╚═╝░░░░░╚═╝╚══════╝
-        """
+# Add top-level CLI commands
+add_env_command(bentoml_cli)
+add_login_command(bentoml_cli)
+add_model_management_commands(bentoml_cli)
+add_start_command(bentoml_cli)
+add_serve_command(bentoml_cli)
+add_containerize_command(bentoml_cli)
 
-    # Add top-level CLI commands
-    add_env_command(bentoml_cli)
-    add_login_command(bentoml_cli)
-    add_bento_management_commands(bentoml_cli)
-    add_model_management_commands(bentoml_cli)
-    add_start_command(bentoml_cli)
-    add_serve_command(bentoml_cli)
-    add_containerize_command(bentoml_cli)
+bentoml_cli.add_command(bento_commands)
 
-    if psutil.WINDOWS:
-        import sys
+# Bento commands should be accessible from the top level, e.g. 'bentoml list' should call 'bentoml bentos list'
+cli = click.CommandCollection(sources=[bentoml_cli, bento_commands])
 
-        sys.stdout.reconfigure(encoding="utf-8")  # type: ignore
+if psutil.WINDOWS:
+    import sys
 
-    return bentoml_cli
-
-
-cli = create_bentoml_cli()
+    sys.stdout.reconfigure(encoding="utf-8")  # type: ignore
 
 
 if __name__ == "__main__":
-    cli()
+    bentoml_cli()
```

Comment:
we will probably need to do two things here:
1. update the cli_events_map for our tracking
2. update the project_entrypoint in project.toml

Reply:
Oh, this is actually a refactor error, let me revert that...